src/PhpSessionPersistence.php

```diff
@@ -25,6 +25,10 @@
 use function session_name;
 use function session_start;
 use function session_write_close;
+use function sprintf;
+use function gmdate;
+use function time;
+use function filemtime;
```

Comment:
Import statements should be alphabetized (simplifies finding declarations, and reduces chances of duplication).

Reply:
sorry, that's ignorance on my part...I always used to set the order based on which func was called first